### PR TITLE
refactor: remove `killable`

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?
No

### Motivation / Use-Case

less deps

### Breaking Changes
No

### Additional Info
No

### CODE REVIEW

File: lib/Server.js [hunk 1]
```diff
@@ -61,6 +60,31 @@ class Server {
...
+      server.close(cb);
+      sockets.forEach((socket) => {
+        socket.destroy();
+      });
+      // reset so the server can be restarted
+      sockets = [];
+    };
+
+    return server;
+  }
```
Comment (alexander-akait) [inline lib/Server.js:86]: Let's avoid extra method and put this code inside close, no need for this

File: lib/Server.js [hunk 2]
```diff
@@ -1738,6 +1735,27 @@ class Server {
...
+    });
+
+    this.server.kill = (cb) => {
+      this.server.close(cb);
+      sockets.forEach((socket) => {
+        socket.destroy();
+      });
+      // reset so the server can be restarted
+      sockets = [];
+    };
```
Comment (alexander-akait) [inline lib/Server.js:1757]: Move this to close, we don't need this here

File: lib/Server.js [hunk 3]
```diff
@@ -1738,6 +1735,27 @@ class Server {
...
+
+    this.server.on("connection", (socket) => {
+      // add socket to list
+      sockets.push(socket);
+
+      socket.once("close", () => {
+        // remove socket from list
+        sockets.splice(sockets.indexOf(socket), 1);
+      });
+    });
```
Comment (alexander-akait) [inline lib/Server.js:1748]: Move this where we create `server`

File: lib/Server.js (hunk 2 above)
Comment (snitin315) [reply lib/Server.js:1757]: Here? https://github.com/webpack/webpack-dev-server/blob/ae8c523854700d63baca335312ccc55df4bfbc42/lib/Server.js#L1790

**Pull request merged**